administrator/components/com_cookiemanager/tmpl/consent/edit.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_cookiemanager
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+\defined('_JEXEC') or die;

```

Comment:
Qualifier is unnecessary and can be removed. You only need a `\` before `defined` when in a file with a namespace 

```suggestion
defined('_JEXEC') or die;

```